src/Selenium2Library/keywords/_element.py

```diff
@@ -480,13 +480,16 @@ def press_key(self, locator, key):
         """Simulates user pressing key on element identified by `locator`.
 
         `key` is either a single character, or a numerical ASCII code of the key
-        lead by '\\'. In test data, '\\' must be escaped, so use '\\\\'.
+        lead by '\\\\'. In test data, '\\' must be escaped, so use '\\\\'.
 
         Examples:
-        | Press Key | text_field   | q |
-        | Press Key | login_button | \\\\13 | # ASCII code for enter key |
+        | Press Key | text_field   | q                |                                               |
+        | Press Key | login_button | \\\\13           | # ASCII code for enter key                    |
+        | Press Key | nav_console  | \\\\\\\\ARROW_UP | # selenium.webdriver.common.keys ARROW_UP KEY |
```

Comment:
The escapes were added to be correctly seen in HTML pages and RIDE context help.
They are double backslash for ASCII codes, and quadruple for KEY NAME, and then escaped for being shown.